acceptance/runAPIacceptance.sh

```diff
@@ -0,0 +1,211 @@
+#!/bin/bash
+#
+# Runs the docker container in which cucumber executes.
+# See usage statement below for more details
+#
+# NOTE: To pass options to cucumber, you must set the CUCUMBER_OPTS
+#       environment variable. For example,
+#       $ CUCUMBER_OPTS="--name MyFeature" ./dockerRun.sh
+#
+
+#
+# Set defaults
+#
+debug=false
+interactive=false
+DRIVER_NAME=selenium_chrome
```

Comment:
`DRIVER_NAME` is only used by the UI tests, so it can be removed from this test harness